spec/enumerable_spec.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require File.expand_path('../enumerable_methods', __dir__)
+
+RSpec.describe do
+  describe '#my_each' do
+    let(:array) { [1, 2, 3, 4] }
+    let(:new_array) { [] }
```

Comment:
Instead of redefining this inside every describe block, let's make them global variables.